scheds/rust/scx_loader/meson.build

```diff
@@ -0,0 +1,9 @@
+sched = custom_target('scx_loader',
+              output: '@PLAINNAME@.__PHONY__',
+              input: 'Cargo.toml',
+              command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
+                        cargo_build_args],
+              env: cargo_env,
+              depends: sched_deps,
+              build_by_default: false,
+              build_always_stale: true)
```

Comment:
If you pull in the latest, `scheds/rust/meson.build` has an example of how to hook this up from the parent directory. I think we just need to add the same template to `rust/meson.build`, specify `scx_loader` as the only target (as others are libraries) and make the top level `meson.build` to descend into the `rust` directory.

Reply:
I'm tried to add that. seems to work)

Although installation of dbus spec is a bit messy :)